src/ontology/mondo-ingest.Makefile

```diff
@@ -616,14 +616,15 @@ EXTERNAL_FILES = \
 # postprocessing steps needed, like Animal QC (which relies on the rare subset pipeline,
 # which is SPARQL postprocessing based).
 
+# TODO: Determine if DOWNLOAD_EXTERNAL=true here is really what is needed. Currently if set to false, will error in `mondo` simulation because it references a file that doesn't exist.
 $(TMPDIR)/mondo-with-simulated-emc-run.owl: $(TMPDIR)/mondo_repo_built $(foreach n,$(EXTERNAL_FILES), $(EXTERNAL_CONTENT_DIR)/$(n).robot.owl)
 	mkdir -p $(TMPDIR)/mondo/src/ontology/tmp/external/
 	$(foreach n,$(EXTERNAL_FILES), cp -f $(EXTERNAL_CONTENT_DIR)/$(n).robot.owl $(TMPDIR)/mondo/src/ontology/tmp/external/processed-$(n).robot.owl;) \
 	cd $(TMPDIR)/mondo/src/ontology && \
 	touch mondo-edit.owl && \
 	cp mondo-edit.owl mondo-edit.owl.bak && \
 	cp mondo-edit.obo mondo-edit.obo.bak && \
-	make update-external-content-incl-rare -B MIR=false IMP=false MIR=false DOWNLOAD_EXTERNAL=false &&\
+	make update-external-content-incl-rare -B MIR=false IMP=false MIR=false DOWNLOAD_EXTERNAL=true &&\
```

Comment:
This is wrong, please do not make this change. 

The error happens because this file has been removed from the pipeline 1 day before I went on paternity leave; the correct solution is to get rid of mentions of that file in the Mondo pipeline, i.e. https://github.com/monarch-initiative/mondo/blob/325f26c658ed406f1df79f0d32402a715a851eab/src/ontology/mondo.Makefile#L457 and subsequent lines.